components/k8s-dashboard-proxy/README.md

```diff
@@ -0,0 +1,34 @@
+# Kubernetes Dashboard Proxy
+
+## Overview
+
+The `Kubernetes Dashboard Proxy` is a transparent proxy for the Kubernetes Dashboard to enable Kyma clients to control the access to the `Kubernetes Dashboard`. `Kubernetes Dashboard` by default is accessible with no authentication/authorization required. `Kubernetes Dashboard Proxy` leverages the Kyma Identity Provider (dex) to authenticate and authorize the `Kubernetes Dashboard` users.
+
+## Concept
```

Comment:
Thanks!
As we discussed we will leave it as a concept because it describes the internal mechanics way more than how to use it. Usage is basically on the other side of kyma, i.e. The Helm Chart as this is basically what kyma users will use/configure. ✌️

Reply:
Ok, let's leave it as it is. I will talk to the writers anyways to figure out what to do in such situations.